primitives/consensus/aura/src/lib.rs

```diff
@@ -24,7 +24,6 @@ use sp_runtime::ConsensusEngineId;
 use sp_std::vec::Vec;
 
 pub mod digests;
-pub mod inherents;
```

Comment:
I'm not seeing this file being removed; did you forget to `git add` the removal? (:

Reply:
Good catch, thanks.